offchain-modules/packages/x/src/db/ckb.ts

```diff
@@ -119,8 +119,8 @@ export class CkbDb {
     }
   }
 
-  async updateCkbMintStatus(mintTxHash: string, status: dbTxStatus) {
-    return this.connection
+  async updateCkbMintStatus(mintTxHash: string, status: dbTxStatus): Promise<void> {
+    this.connection
```

Comment:
`return this.connection...` or `await this.connection...`, otherwise `await updateCkbMintStatus()` will continue to run without waiting for the update to finish